### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Busiwa24**
In the following pull request, we have created a contact form that ensures that:

-the name field accepts up to 30 chars.
-email field accepts only emails.
-message field accepts up to 500 chars.
-all fields have placeholder text.
-all fields have the correct type.
-the user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,166 +1,256 @@
...
+                Get Started
+            </h1>
+            <p class="par-1">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+             Unde ducimus corporis asperiores eveniet architecto, eos laborum velit quasi.
+             Debitis corporis provident cum nihil officiis recusandae explicabo consequuntur suscipit,
+             reiciendis enim!
+            </p>
+        </div> 
+        <div class="form-container">
+            <form action="action_page.php" class="form">
```
Comment (rloterh) [inline index.html]: - [x] Your form is not being submitted successfully. We actually want data to be collected by using the [Formspree service](https://formspree.io/html) by making the form `action`'s value to be similar to `https://formspree.io/f/{form_id}`. Kindly visit [formspree.io](https://formspree.io) and follow the instructions on how to get started, and then update the form action appropriately.
- [x] Also, kindly ensure that this form has a `method` attribute with  value equals `POST`

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Busi and Rustam, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Good UI design
- No linter errors


## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] 
Kindly ensure to make your ReadMe more descriptive by providing a more meaningful title. 

- [OPTIONAL] 
You can improve the `Authors` section of your ReadMe by adding the details of the other collaborator _Rustam_ of this project.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Busiwa24) [reply index.html]: Thanks for the feedback, the issue has been resolved you can have a look

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**